db/app/controllers/api.controller.js

```diff
@@ -0,0 +1,9 @@
+const { fetchAPI } = require("../models/api.models");
+
+exports.getAPI = (req, res, next) => {
+  fetchAPI()
+    .then((apiDetails) => {
+      res.status(200).send({ apiDetails });
+    })
+    .catch(next);
```

Comment:
Ideally the catch wants to be catching and error and then invoking next with the error:
```
.catch((err) => {
next(err)
})
```